docs/application/dotnet/guides/nui/Palette.md

```diff
@@ -0,0 +1,105 @@
+# Palette
+
+The Palette APIs support in selecting colors from images to help you create visually engaging apps.
+
+You can use the Palette APIs to design layout themes and apply custom colors to visual elements in your application. For example, you can use a Palette to create a color-coordinated title card for a song based on its album cover or to adjust an apps toolbar color when its background image changes. The Palette object gives you access to the colors in a bitmap image, while also providing six main color profiles from the bitmap to help your design choices.
+
+## Add namespace
+
+To implement Palette, include `Tizen.NUI` namespace in your application:
+
+```csharp
+using Tizen.NUI;
+```
+
+## Create Palette
+
+Generate a Palette instance using the `ImageLoading` to first create a `PixelBuffer` from an Image.
+
+1. Create `PixelBuffer` from source image:
+
+    ```csharp
+    PixelBuffer imgBitmap = ImageLoading.LoadImageFromFile(imagePath);   
+    ```
+   - You can generate the Palette either synchronously or asynchronously.
```

Comment:
@mijin85cho this bullet list does not seem well aligned in staging.
We could also remove this line as the same info is repeated in next step.